src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -1612,6 +1613,16 @@ public SyntaxNode DottedName(string dottedName)
         public abstract SyntaxNode TypeExpression(ITypeSymbol typeSymbol);
 
         /// <summary>
+        /// Creates an expression that denotes a type.
+        /// Adds a <see cref="DoNotAddImportsAnnotation"/> which will prevent any
```

Comment:
The SymbolAnnotation is needed so that the simplifier can _remove_ `System` if the using is present.